<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="historique">
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>1000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Historique :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="undo">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="gridRank"/>
        </item>
        <item row="4" column="0">
         <layout class="QGridLayout" name="gridTest"/>
        </item>
        <item row="5" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="maximumSize">
           <size>
            <width>1920</width>
            <height>1080</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1269</width>
             <height>69</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1000</height>
            </size>
           </property>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>851</width>
              <height>486</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="topMargin">
              <number>30</number>
             </property>
             <property name="spacing">
              <number>1</number>
             </property>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QPushButton" name="valider">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>VALIDER L'ORDRE</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_eval">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Évaluez la différence entre les deux profils :</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="suite">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>AFFICHER LES RESULTATS</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignHCenter">
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="iconSize">
           <size>
            <width>500</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QPushButton" name="quit">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>QUITTER</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QPushButton" name="save">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>SAUVEGARDER LES RESULTATS</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridCompa">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
         </layout>
        </item>
        <item row="16" column="0">
         <widget class="QPushButton" name="groupeSuivant">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GROUPE SUIVANT</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Réorganisez les profils du meilleur au pire.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="Line" name="line_2">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelValeur">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QLabel" name="text_1">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>0</y>
          <width>881</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vous allez être amenés à classer 20 profils de joueurs évalués selon quatre critères de performance.</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>911</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Les profils sont proposés dans un ordre aléatoire. Pour modifier le classement d'un profil, 
cliquez dessus et faites le glisser sur le profil à droite de l'emplacement désiré.</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>1071</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Les profils sont à classer du meilleur au pire. 
Une fois le classement effectué, il vous faudra choisir l'intensité de votre différence de préférence entre deux profils consécutifs.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cont">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>450</y>
          <width>330</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Continuer</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>27</x>
          <y>300</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QProgressBar::chunk {background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(97, 97, 97, 117));width: 9px;font-size: 18px;}
</string>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="text_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>991</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Les critères sont qualitatifs et représentés par des jauges : plus la jauge est remplie, meilleures sont les performances.</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>340</y>
          <width>1081</width>
          <height>60</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Par exemple, si la jauge associée au critère &quot;fréquence de blessures&quot; est remplie, cela signifie que le joueur se blesse rarement.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="exProfil">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>181</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>180</width>
          <height>115</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="labelExProfil">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Exemple de profil</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_crit1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>297</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>FREQUENCE DE BLESSURES</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>360</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar::chunk {background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(97, 97, 97, 117));width: 9px;font-size: 18px;}</string>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_crit2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>357</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>NOMBRE DE BUTS</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QPushButton" name="begin">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>190</y>
          <width>241</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Commencer</string>
        </property>
       </widget>
       <widget class="QLabel" name="title">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>30</y>
          <width>651</width>
          <height>121</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>20</pointsize>
          <underline>false</underline>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Élicitation d'un modèle de préférences</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>140</y>
          <width>321</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Charger des données</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>130</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="labelNbGroupes">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>100</y>
          <width>171</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre de groupes</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_9">
       <widget class="QLabel" name="text_6">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>100</y>
          <width>1181</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A présent, vous allez comparer deux à deux les profils que vous venez d'ordonner. 
Vous avez le choix entre 4 catégories sémantiques de différence d'attracivité: 
aucune différence  - différence faible - différence intermédiaire - différence forte.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cont2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>300</y>
          <width>330</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Continuer...</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1317</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuitter"/>
    <addaction name="actionInstructions"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>Réinitialiser</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
